pdc/apps/package/tests.py

```diff
@@ -123,9 +123,15 @@ def test_query_with_params(self):
         response = self.client.get(url + '?srpm_nevra=null', format='json')
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertEqual(response.data.get('count'), 1)
+
         response = self.client.get(url + '?compose=compose-1', format='json')
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertEqual(response.data.get('count'), 2)
+        results = response.data.get('results')
```

Comment:
Better to provide a default value '[]' for it, since `results` will be iterated in following code.